app/scripts/modules/ecs/src/serverGroup/configure/#serverGroupConfiguration.service.ts#

```diff
@@ -0,0 +1,497 @@
+import { module, IPromise, IQService } from 'angular';
```

Comment:
What is this file?  It looks to be a dupe of `app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts` but with pound signs in the filename?

Reply:
Looks like an Emacs buffer auto-save... :)

Reply:
Whoops you are right, removed

Reply:
I didn't know.  I use vim, like a normal person.  ;)